Sources/CoreAardvark/Logging/ARKExceptionLogging.m

```diff
@@ -50,12 +51,25 @@ void ARKHandleUncaughtException(NSException *_Nonnull exception)
     NSLock *const logDistributorsLock = ARKGetUncaughtExceptionLogDistributorsLock();
     [logDistributorsLock lock];
 
+    dispatch_group_t observerGroup = dispatch_group_create();
```

Comment:
is it possible for us to add unit tests to this method?

Reply:
I added a unit test in 454d943. I'm not sure a great way to test the timing of it, but the test at least confirms that the method is called.

Reply:
I think that's reasonable!